types/utils.go

```diff
@@ -39,3 +42,20 @@ func parseCSV(csvFilePath string, questionBank *QuestionBank) error {
 	return nil
 }
 
+func clearScreen() {
+	os := runtime.GOOS
+	// if linux or mac`
+	if (os == "linux" || os == "darwin"){
+		cmd := exec.Command("clear")
+		cmd.Stdout = os.Stdout
+		cmd.Run()
+	} else if (os == "windows") {
+		cmd := exec.Command("cls")
+		cmd.Stdout = os.Stdout
+		cmd.Run()
+	} else {
+		log.Fatal("Unsupported Operating System! Can't clear your screen")
+	}
+	return nil
+}
```

Comment:
Early disclaimer: I'm not a Go expert :)

I would make this function return an error, and also avoid the duplicated statements `cmd.Stdout = os.Stdout` & `cmd.Run()` in the two if blocks. Something like:

```Go
func clearScreen() error {
	os := runtime.GOOS

	var cmdStr string
	switch (os) {
	case "linux", "darwin":
		cmdStr = "clear"
	case "windows":
		cmdStr = "cls"
	default:
		// contruct err here with a helpful message (maybe also log it).
		return err
	}

	cmd := exec.Command(cmdStr)
	cmd.Stdout = os.Stdout
	cmd.Run()

	return nil
}
```

WDYT?

Reply:
This looks clean. But the problem is that the function is not doing anything so refactoring it won't help I guess.